src/vuestic-theme/vuestic-components/vuestic-tabs/VaTabs.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="va-tabs">
+    <div
+      class="va-tabs__bar"
+      :class="{
+       'align-right': right,
+       'grow': grow
+      }"
+    >
+      <div
+        class="va-tabs__bar-content"
+        :class="{'grow': grow}"
+        :style="{'background-color': color}"
```

Comment:
Use one naming for properties in style objects: 'background-color' -> backgroundColor.
Fix there and below.